### PR TITLE
Not return unimplemented capabilities

### PR DESCRIPTION
**Pull request by EvgenyUglov**
# Description
PR removes STAGE_UNSTAGE_VOLUME unimplemented node capability from the returned list

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1559|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] make unit-test
- [x] Installed PowerStore driver and checked the list of capabilities
```
[root@master-1-AevzcJKPoLLnD samples]# kubectl logs isilon-node-z5dkt -n isilon | grep STAGE
[root@master-1-AevzcJKPoLLnD samples]# kubectl logs isilon-node-z5dkt -n isilon | grep rpc
time="2024-11-22T17:45:22Z" level=info msg="/csi.v1.Node/NodeGetCapabilities: REP 0003: Capabilities=[rpc:<type:SINGLE_NODE_MULTI_WRITER >  rpc:<type:GET_VOLUME_STATS >  rpc:<type:VOLUME_CONDITION > ], XXX_NoUnkeyedLiteral={}, XXX_sizecache=0"
```

### CODE REVIEW

File: service/node.go [hunk 1]
```diff
@@ -286,20 +286,22 @@ func (s *service) NodeGetCapabilities(
 	*csi.NodeGetCapabilitiesResponse, error,
 ) {
 	capabilities := []*csi.NodeServiceCapability{
-		{
+		// TODO - Need to return this capability when Stage/Unstage volume logic is implemented
+		/*{
```
Comment (santhoshatdell) [inline service/node.go:290]: I think we can remove the unsupported capabilities instead of keeping them commented.

Comment (EvgenyUglov) [reply service/node.go]: Removed. Thanks!

**Review by santhoshatdell — APPROVED** [review]

**Comment by santhoshatdell** [comment]
@EvgenyUglov Could you rebase the branch? All checks should pass now.

**Comment by EvgenyUglov** [comment]
> @EvgenyUglov Could you rebase the branch? All checks should pass now.

Done

**Review by jackieung-dell — APPROVED** [review]

**Pull request merged**